test/sequential/test-inspector-async-call-stack-abort.js

```diff
@@ -31,7 +30,7 @@ if (process.argv[2] === 'child') {
   const options = { encoding: 'utf8' };
   const proc = spawnSync(
     process.execPath, ['--expose-internals', __filename, 'child'], options);
-  strictEqual(proc.status, 0);
+  strictEqual(proc.status, 1);
```

Comment:
Why is this changed?

Reply:
This code spawns itself and then lets itself die (with `throw new Error(eyecatcher)`), it used to emit a warning (on unhandled rejection) and then terminate (with status 0, since Node didn't crash on unhandled rejections).

Now it emits the error and then dies - so it terminates with status 1.

I am not sure why the original code suppressed unhandled rejections though since that would still test that no process segfault occurs. Looking at the [blame](https://github.com/nodejs/node/commit/df08779e0dfa82ceb69e129ae0548e97352aaddf) this looks mostly for historical reasons since tests didn't use to crash on unhandled rejections by default.